docs/source/configuration/overview.mdx

```diff
@@ -578,6 +578,20 @@ The default value of `experimental_parallelism` is `1`.
 
 In practice, you should tune `experimental_parallelism` based on metrics and benchmarks gathered from your router.
 
+<MinVersion version="1.51.0">
+
+### Introspection response caching
+
+Introspection responses are generated by the query planner for now, so they are expensive to execute and the router stores them in its query planner cache. Unfortunately, they can fill up the cache, so until we move out introspection execution, there is an option to deactivate response caching.
+
+```yaml title="router.yaml"
+supergraph:
+  query_planning:
+    legacy_introspection_caching: true
```

Comment:
```suggestion
    legacy_introspection_caching: false
```

The text avove talks about deactivating introspection caching, but then the example shows it being enabled.